sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/V1Writes.scala

```diff
@@ -43,7 +43,7 @@ trait V1WriteCommand extends DataWritingCommand {
   /**
    * Specify the partition spec of the V1 write command.
    */
-  def staticPartitions: TablePartitionSpec
+  def statementStaticPartitions: TablePartitionSpec
```

Comment:
Why do we need to rename this function? Has its definition changed?

Reply:
For ease of understanding, there is no need to rename the function. It has been changed back.